### PR TITLE
docs: added basic docsify documentation

### PR DESCRIPTION
**Pull request by Alex-Werner**
### Issue being fixed or implemented  

In order to refer to DAPI-Client in order documentation (see this open PR : https://github.com/dashevo/wallet-lib/pull/100), and in order to provide an easy overlook at the features enabled by DAPI-Client, we needed a very basic usage documentation. 
This PR brings the same documentation experience we have on our other repository (usage + function doc + issue/pr template). 

### What was done  

- chore: docsify (no jekyll)
- docs: Application, Payment and Utils method
- chore: Github template

### Notes 
After this PR is merged, it will still requires to activate under the settings our docs folder (for github.io access) and we will need to modify the repo configuration to add the github.io link.

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      noData: {
+        '/': 'No results!'
+      },
+      paths: 'auto',
+      placeholder: {
+        '/': 'Search'
+      }
+    },
+    // search: 'auto',
+    formatUpdated: '{MM}/{DD} {HH}:{mm}',
```
Comment (thephez) [inline docs/index.html]: Add Dash blue theme color
```suggestion
    formatUpdated: '{MM}/{DD} {HH}:{mm}',
    themeColor: '#008de4',
```

File: docs/_sidebar.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    - Payment
+        - [`.getBestBlockHash()`](usage/payment/getBestBlockHash.md)
+        - [`.getBlockHash()`](usage/payment/getBlockHash.md)
+        - [`.getUTXO()`](usage/payment/getUTXO.md)
+   - Utilitaries
+        - [`.getGrpcUrl()`](usage/utils/getGrpcUrl.md)
+        - [`.getMnListDiff()`](usage/utils/getMnListDiff.md)
+        - [`.subscribeToTransactionsWithProofs()`](usage/utils/subscribeToTransactionsWithProofs.md)
+- Examples
+    - [RegisterUser](https://github.com/dashevo/dapi-client/blob/master/examples/registerUser.js)
```
Comment (thephez) [inline docs/_sidebar.md]: This example is terribly out of date. Remove from this until it is updated?

File: docs/getting-started/quickstart.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+(async () => {
+  const bestBlockHash = await client.getBestBlockHash('testaddress');
+  console.log(bestBlockHash);
+})();
+```
+
+## Quicknotes
+
+This package allows you to fetch & send informations from both the payment chain (layer 1) and the application chain (layer 2, a.k.a Platform chain).
```
Comment (thephez) [inline docs/getting-started/quickstart.md]:
```suggestion
This package allows you to fetch & send information from both the payment chain (layer 1) and the application chain (layer 2, a.k.a Platform chain).
```

File: docs/README.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+## Wallet-lib
```
Comment (thephez) [inline docs/README.md]: Update for DAPI-client (copy/pasted from Wallet-lib)?

File: docs/_sidebar.md (hunk 2 above)
Comment (Alex-Werner) [reply docs/_sidebar.md]: Agreed, it has been removed for now :) Will wait a little bit for dapi-client to return stability and I will update this one.

File: docs/README.md (hunk 4 above)
Comment (Alex-Werner) [reply docs/README.md]: Updated !

**Comment by Alex-Werner** [comment]
What are your toughts @thephez with the current moving state of DAPI-Client ? 
Would we merge this now, and update in another PR (ideally the PR that adds the feature)
Or do we wait for this to stabilize, update in this PR and merge ?

**Comment by thephez** [comment]
@Alex-Werner I think I would rather merge now and then fix later. That way the PR size does not get huge and overwhelming.

**Pull request merged**